src/PKSim.Assets/PKSimConstants.cs

```diff
@@ -906,6 +906,9 @@ public static string MapToSnapshotNotSupportedWithoutContext(string snapshotType
 
          public static string CannotFindSimulationInSnapshot(string simulationName, string project) => CannotFindBuildingBlockInSnapshot(ObjectTypes.Simulation, simulationName, project);
 
+         public static string SimulationUsedInPlotsAreNotExported(IReadOnlyList<string> simulationNames, string project)
```

Comment:
Making sure that simulation used in plots are also referenced here. This is just a simple validation step.